MDPSplitView.m

```diff
@@ -45,20 +45,24 @@ static NSInteger MDPKeyToIndex(NSString *key)
     return [formatter numberFromString:[key substringFromIndex:MDPKeyPrefix.length]].integerValue;
 }
 
-@implementation MDPSplitView
+@interface MDPSplitView ()
 
-#pragma mark - NSView
+/*!
+ The number of active animations for each divider by index.
+ */
+@property (strong, nonatomic) NSCountedSet *mdp_animationCounts;
 
-+ (id)defaultAnimationForKey:(NSString *)key
+@end
+
+static void CommonInit(MDPSplitView *self)
```

Comment:
That's a great idea!

But since you pointed out that I don't need synchronization, I may make the accessor lazy instead. :sparkles: